crates/volta-core/src/error/details.rs

```diff
@@ -590,13 +595,28 @@ Please remove the file or pass `-f` or `--force` to override.",
 {}",
                 REPORT_BUG_CTA
             ),
+            #[cfg(feature = "volta-updates")]
+            ErrorDetails::CouldNotStartMigration => write!(
+                f,
+                "Volta is out-of-date and could not start migration process.
+
+Please ensure Volta was installed correctly."
```

Comment:
I like the idea of asking them to directly run `volta-migrate`. That actually is probably the highest likelihood of success, since the issue is that `volta-migrate` wasn't able to start, which most often means it's either not there or it's there and not executable. Both would need to be fixed in order to run it, so that's a good CTA!